02-library/cs506/sim.py

```
@@ -1,13 +1,64 @@
+error_msg = "There is an error in input."
+
+
+## NOTE: Used the pull request from RC-88 to make this file - all credit goes to them.
+## Was implementing the kmeans file and wasn't sure if that would be dependent on this one.
+## so I used their implementations. Only difference made was creating the error msg as a constant.
+
+
 def euclidean_dist(x, y):
-    raise NotImplementedError()
+    if x == [] and y == []:
+        return error_msg
```

it's better practice to raise an error instead of returning an inconsistent type.